src/html/header-basic.html

```diff
@@ -1,6 +1,40 @@
 
   <a class="usa-skipnav" href="#main-content">Skip to main content</a>
   <header class="usa-header" role="banner">
+    <!-- Gov banner BEGIN -->
```

Comment:
Do you mean are we adding a comment anywhere else in our examples?
No, we're not, but it might be a good practice to do so for complex components made up of many components. It could help add more clarity to the code. I appreciate how Google Material Design Lite does this: https://getmdl.io/components/index.html#layout-section

> They might wonder if the skipnav above this line is also part of our example code.
Since it all gets added to the code sample, I think they'll be able to see the entire snippet and know it's all together. 

Additionally, we've not heard that concern from users before. We've actually heard the opposite, that it's hard to read/confusing when the snippets are too large (at least that's what I remember hearing from the listening tour).